org.eclipse.ice.archetypes/data-element-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<groupId>org.eclipse.ice</groupId>
+		<artifactId>data-renderer-starter</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>${groupId}</groupId>
+	<artifactId>${artifactId}</artifactId>
+	<version>${version}</version>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.11</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
```

Comment:
If I'm looking at the right file, don't we want to include `org.eclipse.ice.dev.annotations` in the dependencies here?

Reply:
Just saw your comments in your OP -- seems I might be off. Which pom file is preserved as the parent of projects generated from the archetype?

Reply:
Yes, we do.

Reply:
The top level pom for the archetypes is set as the parent, which is where I put the annotations dependency. I just did that so, if that dependency needed to be changed, you could change it in one place and every archetype would just pull from that, but maybe it would be better to just put it inside the actual pom template.